CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+0.4.3 (to be released)
+----------------------
+
+* Allowed retry policies to be specified as import path strings, which is
+  required for the ``ZYTE_API_RETRY_POLICY`` setting, and allows requests with
+  the ``zyte_api_retry_policy`` request.meta key to remain serializable.
+* Zyte Data API error responses (after retries) are no longer ignored, and
+  instead raise a ``zyte_api.aio.errors.RequestError`` exception, which allows
+  user-side handling of errors and provides better feedback for debugging.
```

Comment:
I think this is atechnically backwards incompatible changes, which warrants a 0.5.0 release